log/log_test.go

```diff
@@ -39,6 +42,71 @@ func capture(f func()) string {
 	return buf.String()
 }
 
+var logLevelEnvVarName = "INSOLAR_LOG_LEVEL"
+
+func testWithEnvVar(t *testing.T) {
+	val := strings.ToLower(os.Getenv(logLevelEnvVarName))
+	fmt.Printf("testWithEnvVar: %v=%v\n", logLevelEnvVarName, val)
+
+	assert.Containsf(t,
```

Comment:
Do we need here exactly 'assert'? Might be it's better to use 'require'

Reply:
it's ok, the test still fails with assert